integreat_cms/cms/templates/events/event_list_row.html

```diff
@@ -5,10 +5,10 @@
 
 <tr class="border-t border-solid border-gray-200 hover:bg-gray-100 text-gray-800">
     <td class="py-3 pl-4">
-        <input type="checkbox" name="selected_ids[]" value="{{ event.id }}" form="bulk-action-form" class="bulk-select-item">
+        <input type="checkbox" name="selected_ids[]" value="{{ event.id }}" form="bulk-action-form" class="bulk-select-item" data-bulk-select-item/>
```

Comment:
```suggestion
        <input type="checkbox" name="selected_ids[]" value="{{ event.id }}" form="bulk-action-form" class="bulk-select-item" />
```
I think it's redundant to add a data-attribute for all elements which already have the class `bulk-select-item`.